### DISCUSSION TITLE
throwErrors/useErrorBoundary for Vue

### DISCUSSION THREAD

**Discussion by rogierpennink** [TanStack/query#4926] [Q&A] [answered]
Hello!

First, thanks for this library, it's proven really useful during my transition from Vue2 to Vue3. I did run into one issue though. According to the documentation, it should be possible to provide a `useErrorBoundary` in the mutation options when using `useMutate()`, which I expected would cause any errors in the mutation function to simply bubble up. This would be great, because I implemented my own ErrorBoundary component and would like all errors to be processed at that level.

Unfortunately, I cannot get this to work. Is there something I'm missing?

**Comment by louis-young** [comment, marked as answer]
Hi 👋 

I'm not a Vue user but it'd be helpful if you could provide a minimal runnable reproduction. It's hard to reason about why it might not be working/what you might be missing without seeing what you're trying to do 🙂

**Reply by rogierpennink** [reply]
Apologies, turns out it works just fine, but one of my Axios interceptors was not handling the error the right way!

**Reply by louis-young** [reply]
No problem 🙂

**Comment by arnoson** [comment]
@rogierpennink, can you describe how exactly your setup is?
I wan't to catch an error in a `NuxtErrorBoundary`, but it's not working
```ts
// Inside the setup script of my component, that is wrapped in a `NuxtErrorBoundary`

const mutation = useMutation(
  () => {
    throw new Error('oh no!')
  },
  { useErrorBoundary: true }
)

mutation.mutate()
```

**Comment by arnoson** [comment]
See this [codesandbox](https://codesandbox.io/p/github/arnoson/tanstack-vue-error-boundary/main?workspaceId=d6d5c274-4380-448b-847c-0327b5391d0a)
Maybe I am missing something really obvious?
